tests/system/Makefile

```diff
@@ -0,0 +1,27 @@
+PREFIX ?= riscv32-unknown-elf-
+ARCH = -march=rv32izicsr
```

Comment:
Should it be `rv32i_zicsr` instead?

Reply:
> Should it be `rv32i_zicsr` instead?

Yes, given that we're using the xPack toolchain in the CI environment, the `rv32i_zicsr` flag will be the correct flag once the test suite is integrated.